src/renderer/components/+preferences/preferences.tsx

```diff
@@ -111,6 +120,51 @@ export class Preferences extends React.Component {
     );
   }
 
+  renderEditorConfiguration(editorType: EditorType) {
+    if (this.activeEditorType == editorType)  
+    {return (<>
+      <section>
+        <FormSwitch
+          control={
+            <Switcher
+              checked={UserStore.getInstance().editorConfiguration.get(editorType).miniMap.enabled}
+              onChange={v => UserStore.getInstance().editorConfiguration.get(editorType).miniMap.enabled = v.target.checked}
+              name="minimap"
+            />
+          }
+          label="Show minimap"
+        />
+      </section>
+      <section>
+        <FormSwitch
+          label="Line numbers"
+          control={ <Select
+            options={[{label: "On", value: "on"}, 
+              {label: "Off", value: "off"}, 
+              {label: "Relative", value: "relative"}, 
+              {label: "Interval", value: "interval"}]}
```

Comment:
These options should probably be defined as a const object somewhere else and use an `enum`.